client/src/component/registerForm/body/agreement.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import Box from "@material-ui/core/Box";
+import Checkbox from "@material-ui/core/Checkbox";
```

Comment:
is it possible to import components like?
import { Checkbox, Button, Box } from "@material-ui/core